src/redux-store/store.js

```diff
@@ -0,0 +1,197 @@
+/* eslint-disable complexity */
+import {createStore, applyMiddleware} from 'redux';
+import axios from 'axios';
+import thunk from 'redux-thunk';
+
+const initialState = {
+  categories: [],
+  proucts: [],
+  users: [],
+};
+
+const GOT_CATEGORIES = 'GOT_CATEGORIES';
+const CREATE_CATEGORY = 'CREATE_CATEGORY';
+const UPDATE_CATEGORY = 'UPDATE_CATEGORY';
+const GOT_PRODUCTS = 'GOT_PRODUCTS';
+const CREATE_PRODUCT = 'CREATE_PRODUCT';
+const UPDATE_PRODUCT = 'UPDATE_PRODUCT';
+const GOT_USERS = 'GOT_USERS';
+const CREATE_USER = 'CREATE_USER';
+const UPDATE_USER = 'UPDATE_USER';
+
+const gotCategories = (categories) => {
+  return {
+    type: GOT_CATEGORIES,
+    categories,
+  };
+};
+
+const gotProducts = (products) => {
+  return {
+    type: GOT_PRODUCTS,
+    products,
+  };
+};
+
+const gotUsers = (users) => {
+  return {
+    type: GOT_USERS,
+    users,
+  };
+};
+
+const createCategory = (category) => {
+  return {
+    type: CREATE_CATEGORY,
+    category,
+  };
+};
+
+const createProduct = (product) => {
+  return {
+    type: CREATE_PRODUCT,
+    product,
+  };
+};
+
+const createUser = (user) => {
+  return {
+    type: CREATE_USER,
+    user,
+  };
+};
+
+const updatedCategory = (category, id) => {
```

Comment:
I don't think you need to have the `id` as a parameter since the API route is returning the entire `Category` object.